Bio/SeqFeature.py

```diff
@@ -1669,7 +1772,7 @@ def __repr__(self):
         """Represent the ExactPosition object as a string for debugging."""
         return "%s(%i)" % (self.__class__.__name__, int(self))
 
-    def _shift(self, offset):
+    def __add__(self, offset):
```

Comment:
+1. The use of ``__add__`` here is rather elegant, I wonder if I just missed this idea - or more likely added ``_shift`` before the integer subclassing change had been completed.